migrations/sqlite/20240809205107_event_init_cid_informant_type.up.sql

```diff
@@ -6,4 +6,7 @@ ALTER TABLE ceramic_one_event ADD COLUMN init_cid BLOB DEFAULT NULL;
 -- The ID of the node that sent the event or our own ID if the event came in via API
 ALTER TABLE ceramic_one_event ADD COLUMN informant TEXT DEFAULT NULL;
 
-SELECT init_cid, informant FROM "ceramic_one_event" WHERE false;
+-- Whether this is a Time Event or not
+ALTER TABLE ceramic_one_event ADD COLUMN is_time_event BOOLEAN DEFAULT FALSE;
```

Comment:
I'm pretty sure we can't move a migration after it's been applied anywhere. I think we just need to make a new one and leave the old name.

In terms of this column, maybe we should just add event_type integer and create an enum that's 0/1/2 for init/time/data?

Reply:
done